src/processTargets/processSelectionType.ts

```diff
@@ -40,6 +40,13 @@ export default function (
         selection,
         selectionContext
       );
+    case "url":
+      return processRegexDefinedScope(
+        /(http(s)?:\/\/.)?(www\.)?[-a-zA-Z0-9@:%._\+~#=]{2,256}\.[a-z]{2,6}\b([-a-zA-Z0-9@:%_\+.~#?&//=]*)/g, // taken from https://regexr.com/3e6m0
```

Comment:
Note that the specific regex I chose requires a top-level domain, so urls like `http://localhost:3000` would not be found.

We could definitely refactor that, either now or later. Tbh it would just take me a long time to write that regex 😅

Reply:
Preferably those should work as well but it's not a deal breaker for first version.